packages/dataviews/src/types.ts

```diff
@@ -28,7 +28,15 @@ interface FilterByConfig {
 	isPrimary?: boolean;
 }
 
-type Operator = 'is' | 'isNot' | 'isAny' | 'isNone' | 'isAll' | 'isNotAll';
+type DeprecatedOperator = 'in' | 'notIn';
```

Comment:
@ntsekouras @oandregal are these still operators we support. (We do have code that references them, so I had to add them to the types)

Reply:
The only place we use them is this one https://github.com/WordPress/gutenberg/blob/trunk/packages/dataviews/src/utils.js#L20 for backward compatibility.

I think we should consider removing them at some point before 6.6, though they were part of 6.5 ([in](https://github.com/WordPress/gutenberg/blob/wp/6.5/packages/edit-site/src/components/page-patterns/index.js#L326), [not in](https://github.com/WordPress/gutenberg/blob/wp/6.5/packages/edit-site/src/components/page-pages/index.js#L236)). I presume we can still remove them as dataviews is a bundle package/experimental api?

Reply:
Yes, I think we can. Let's do that in a follow-up.